release-notes.md

```diff
@@ -8,12 +8,26 @@ Here are the release notes for the Experience Platform Mobile SDKs and Adobe Pla
 
 The following updates were made in this release:
 
+**iOS Core 2.3.5**
+
+* Added the support for attach data rules consequence.
+* Added the support for boolean type comparison for `Exist` or `Not Exist` rules condition.
```

Comment:
- Please remove "the" from both bullets.
- Please reword the 2nd bullet as follows:
"Added support for a boolean-type comparison for the `Exist` or `Not Exist` rules condition."